cmd/up.go

```diff
@@ -208,38 +208,28 @@ func getPrimaryPort(ports []PortPayload) PortPayload {
 	return PortPayload{}
 }
 
-func transformComposeToNewShipment(shipmentName string, shipment ComposeShipment, dockerComposeProject project.APIProject) NewShipmentEnvironment {
+func transformComposeToShipmentEnvironment(shipmentName string, shipment ComposeShipment, dockerComposeProject project.APIProject) ShipmentEnvironment {
 
 	//create object used to create a new shipment environment from scratch
-	newShipment := NewShipmentEnvironment{
-		Info: NewShipmentInfo{
-			Name:  shipmentName,
-			Group: shipment.Group,
+	newShipment := ShipmentEnvironment{
+		Name:    shipment.Env,
+		EnvVars: make([]EnvVarPayload, 0),
+		ParentShipment: ParentShipment{
+			Name:    shipmentName,
+			EnvVars: make([]EnvVarPayload, 0),
+			Group:   shipment.Group,
 		},
 	}
 
 	//add shipment-level env vars
-	newShipment.Info.Vars = make([]EnvVarPayload, 0)
-	newShipment.Info.Vars = append(newShipment.Info.Vars, envVar("CUSTOMER", shipment.Group))
-	newShipment.Info.Vars = append(newShipment.Info.Vars, envVar("PROPERTY", shipment.Property))
-	newShipment.Info.Vars = append(newShipment.Info.Vars, envVar("PROJECT", shipment.Project))
-	newShipment.Info.Vars = append(newShipment.Info.Vars, envVar("PRODUCT", shipment.Product))
-
-	//create environment
-	newShipment.Environment = NewEnvironment{
-		Name: shipment.Env,
-		Vars: make([]EnvVarPayload, 0),
-	}
-
-	//add environment-level env vars
-	for name, value := range shipment.Environment {
-		newShipment.Environment.Vars = append(newShipment.Environment.Vars, envVar(name, value))
-	}
```

Comment:
What about env-level envvars? I think we still need those for log shipping.

Reply:
true. I didn't think we added any environment level envVars. I'll add this back.